Lib/fontTools/varLib/__init__.py

```diff
@@ -31,9 +31,22 @@
 from fontTools.varLib import designspace, models, builder
 from fontTools.varLib.merger import merge_tables, Merger
 import warnings
+import re
 import os.path
 from argparse import ArgumentParser
 
+CLASS_VALS = {
+	'Thin': 100,
```

Comment:
Could varLib perhaps take the weight and width values from the [OS/2 table](https://www.microsoft.com/typography/otspec/os2.htm) in the merged fonts? This would be less fragile than parsing camel-cased English names.